delta/plugins/blazegraph/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/blazegraph/routes/BlazegraphViewsRoutes.scala

```diff
@@ -75,7 +75,7 @@ class BlazegraphViewsRoutes(
     JsonLdEncoder.computeFromCirce(ContextValue(contexts.statistics))
 
   def routes: Route =
-    (baseUriPrefix(baseUri.prefix) & replaceUri("views", schema.iri, projects)) {
+    baseUriPrefix(baseUri.prefix) & replaceUri("views", schema.iri, projects) {
```

Comment:
this kind of change yields compilation errors.

Reply:
I've fixed it